### PR TITLE
[Clojure] follow up pr 14531 to fix tests

### PR DESCRIPTION
**Pull request by gigasquid**
## Description ##
This is a follow up PR to https://github.com/apache/incubator-mxnet/pull/14531 which fixed some Clojure tests that were not testing what they thought they were.

This PR resolves the FIXME disabled tests in there

### CODE REVIEW

**Comment by gigasquid** [comment]
@Chouffe would you mind taking a look and reviewing this when you get a chance?

**Comment by Chouffe** [comment]
@gigasquid I will take a look shortly when I get a chance. Thanks for finishing the fixes ^^

File: contrib/clojure-package/test/good-test-ndarray.clj [hunk 1]
```diff
@@ -35,3 +35,4 @@
      ndarray-or-double-or-float
      #{"org.apache.mxnet.MX_PRIMITIVES$MX_PRIMITIVE_TYPE"
        "org.apache.mxnet.NDArray"})))))
+
```
Comment (Chouffe) [inline contrib/clojure-package/test/good-test-ndarray.clj]: #nitpick do we need this extra line?

File: contrib/clojure-package/test/org/apache/clojure_mxnet/module_test.clj [hunk 2]
```diff
@@ -280,9 +281,8 @@
                         :index nil
                         :pad 0}]
       (-> mod
-          (m/forward data-batch))
-      ;; FIXME
-      #_(is (= [(first l-shape) num-class]
+          (m/forward data-batch-2))
```
Comment (Chouffe) [inline contrib/clojure-package/test/org/apache/clojure_mxnet/module_test.clj]: awesome catch!

**Comment by abhinavs95** [comment]
@mxnet-label-bot add [Clojure, test]

File: contrib/clojure-package/test/good-test-ndarray.clj (hunk 1 above)
Comment (gigasquid) [reply contrib/clojure-package/test/good-test-ndarray.clj]: It's in the generator - not needed but we can tackle that in another PR touching that area.

**Comment by gigasquid** [comment]
Thanks for reviewing @Chouffe !

**Pull request merged**